src/blob/index/core.rs

```diff
@@ -64,9 +64,18 @@ where
     for<'a> K: Key<'a>,
 {
     const KEY_SIZE: usize = K::LEN as usize;
-    const BPTREE_ENTRY_SIZE: usize =
+    const BTREE_ENTRY_SIZE: usize =
         size_of::<Vec<u8>>() + MemoryAttrs::<K>::KEY_SIZE + size_of::<Vec<RecordHeader>>();
     const RECORD_HEADER_SIZE: usize = size_of::<RecordHeader>() + MemoryAttrs::<K>::KEY_SIZE;
+    // Each node in BTreeMap contains preallocated vectors of 11 values and 12 edges.
+    // Although count of nodes can't be determined without reimplementing insertion algorithm,
+    // we can use approximation of overhead size added per one key
+    const BTREE_SIZE_MULTIPLIER: usize = (
```

Comment:
This multiplier must be of type `f64`. Integer division will result in loss of precision. According to my calculations, the result of the division is `9.81`, which rounds up to `9` in integer division. So with 100_000 entries, this results in 81 KB of data being left out